templates/android/library/src/main/java/io/appwrite/models/RealtimeModels.kt.twig

```diff
@@ -1,5 +1,6 @@
 package {{ sdk.namespace | caseDot }}.models
 
+import kotlin.collections.Collection
```

Comment:
Is this required?

Reply:
Yep it is now - not sure how it wasn't before, but now without this there is a compilation error stating `Collection` does not require type arguments when used [here](https://github.com/appwrite/sdk-generator/blob/9fdb7bf3c4615f6132da47d4beb7f77dbcb8c8e6/templates/android/library/src/main/java/io/appwrite/models/RealtimeModels.kt.twig#L13), as it was referring to `io.appwrite.models.Collection` instead